flux/mpibind.lua

```diff
@@ -0,0 +1,10 @@
+
+-- initrc to load mpibind plugin into flux. To use, add '-o initrc=mpibind.lua'
+-- to your flux mini run command
+prefix = os.execute ("pkg-config --variable=prefix flux-core")
```

Comment:
The Lua `os.execute()` functions does not return the output of the command, unfortunately. You would have to use `io.popen()` instead.

However, this bit of code isn't necessary anymore (or won't be after proposed changes to the default shell initrc.lua) so it can be dropped.